### PR TITLE
Switch prune to externals

### PR DESCRIPTION
**Pull request by soltysh**
/assign @juanvallejo

### CODE REVIEW

File: pkg/oc/cli/admin/groups/sync/prune.go [hunk 1]
```diff
@@ -53,92 +51,73 @@ var (
...
+	cmd.Flags().StringVar(&o.WhitelistFile, "whitelist", o.WhitelistFile, "path to the group whitelist file")
 	cmd.MarkFlagFilename("whitelist", "txt")
-	cmd.Flags().StringVar(&blacklistFile, "blacklist", whitelistFile, "path to the group blacklist file")
+	cmd.Flags().StringVar(&o.BlacklistFile, "blacklist", o.BlacklistFile, "path to the group blacklist file")
 	cmd.MarkFlagFilename("blacklist", "txt")
 	// TODO(deads): enable this once we're able to support string slice elements that have commas
-	// cmd.Flags().StringSliceVar(&options.Blacklist, "blacklist-group", options.Blacklist, "group to blacklist")
-
-	cmd.Flags().StringVar(&configFile, "sync-config", configFile, "path to the sync config")
+	// cmd.Flags().StringSliceVar(&o.Blacklist, "blacklist-group", o.Blacklist, "group to blacklist")
```
Comment (juanvallejo) [inline pkg/oc/cli/admin/groups/sync/prune.go]: Is this a leftover comment?

Comment (soltysh) [reply pkg/oc/cli/admin/groups/sync/prune.go]: Nope, that was here before.

File: pkg/oc/cli/admin/prune/auth/bindings.go [hunk 2]
```diff
@@ -29,7 +32,13 @@ func reapClusterBindings(removedSubject kapi.ObjectReference, c authclient.Inter
 		if len(retainedSubjects) != len(binding.Subjects) {
 			updatedBinding := binding
 			updatedBinding.Subjects = retainedSubjects
-			if _, err := c.Authorization().ClusterRoleBindings().Update(&updatedBinding); err != nil && !kerrors.IsNotFound(err) {
+			coreSubjects, err := convertObjectReference(retainedSubjects)
+			if err != nil {
+				errors = append(errors, err)
+				continue
+			}
+			updatedBinding.UserNames, updatedBinding.GroupNames = authorizationapi.StringSubjectsFor(binding.Namespace, coreSubjects)
```
Comment (soltysh) [inline pkg/oc/cli/admin/prune/auth/bindings.go]: @juanvallejo fyi, this logic, which was previously hidden inside conversions was causing the error, sigh...

**Comment by soltysh** [comment]
Applying labels based on previous approval.

**Pull request merged**